wgpu-hal/src/vulkan/conv.rs

```diff
@@ -74,6 +74,7 @@ impl super::PrivateCapabilities {
                 }
             }
             Tf::Depth16Unorm => F::D16_UNORM,
+            Tf::NV12 => unreachable!(),
```

Comment:
I believe this is `F::G8_B8R8_2PLANE_420_UNORM` according to [ffmpeg](https://github.com/FFmpeg/FFmpeg/blob/ba6a5e7a3d4c372a98f910dda7e4d65847eda817/libavutil/hwcontext_vulkan.c#L194)

Reply:
Currently [ash](https://github.com/ash-rs/ash/blob/f5e7606c039c5a054544231509e1daf436820da5/ash/src/vk/bitflags.rs#L387C6-L387C22) lacks the definition of `VK_IMAGE_ASPECT_PLANE_*_BIT`, I think we can support vulkan at some time later

Reply:
We can still support it later if you want to, but you can define your own flags: https://docs.rs/ash/latest/ash/vk/struct.ImageAspectFlags.html#method.from_raw

Reply:
Done.

Reply:
> Currently [ash](https://github.com/ash-rs/ash/blob/f5e7606c039c5a054544231509e1daf436820da5/ash/src/vk/bitflags.rs#L387C6-L387C22) lacks the definition of `VK_IMAGE_ASPECT_PLANE_*_BIT`, I think we can support vulkan at some time later.

I thought so too, but they are apparently defined here: https://github.com/ash-rs/ash/blob/master/ash/src/vk/feature_extensions.rs#L85-L87